src/components/Sidebar/index.jsx

```diff
@@ -13,27 +19,51 @@ import { currentContextSelector } from 'redux/selectors';
 import 'sweetalert2/dist/sweetalert2.css';
 import styles from './style.scss';
 
-function Sidebar(props) {
-  const annotatedItems = Object.keys(props.context).map(annotationName => (
-    <li key={annotationName}>
-      <AnnotatedItem annotationName={annotationName} />
-    </li>
-  ));
-  return (
-    <div className={styles.sidebar}>
-      <header>
-        <span>{ manifest.name }</span>
-        <span className={styles.saveIndicator} onClick={props.saveAnnotatedItems}>
-          {
-            props.saving
-            ? <Spinner noFadeIn spinnerName={'three-bounce'} />
-            : <MdSave />
-          }
-        </span>
-      </header>
-      <ul className={styles.annotationList}>{annotatedItems}</ul>
-    </div>
-  );
+class Sidebar extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      search: '',
+    };
+    this.searchUpdated = this.searchUpdated.bind(this);
+  }
+
+  getAnnotations() {
+    const annotations = values(mapValues(this.props.context, (annotation, name) => ({
+      ...annotation,
+      name,
+    })));
+    const filteredAnnotations = sortBy(
+      annotations.filter(createFilter(this.state.search, 'name')), 'name');
```

Comment:
Hmm, since the annotations are collapsed by default I feel like it would be weird to search the inner fields. Do you feel strongly?